Client/Frontend/Browser/TabManager.swift

```diff
@@ -740,14 +737,6 @@ extension TabManager: WKNavigationDelegate {
     }
 
     func webView(_ webView: WKWebView, didFail navigation: WKNavigation!, withError error: Error) {
```

Comment:
Since `func webView(_ webView: WKWebView, didFail navigation: WKNavigation!, withError error: Error) {` is now empty, it would be better to remove it. We can reimplement that delegate call when needed.